public/script.js

```diff
@@ -22,8 +22,19 @@ const DEFAULTS = {
 };
 
 let map;
+const validDates = [];
 /** Main function run to start animation */
 async function main() {
+  await fetch('https://nsidc.org/api/mapservices/NSIDC/wms?service=wms&version=1.1.1&request=GetCapabilities').then(function(response) {
+    return response.text();
+  }).then(function(text) {
+    const parser = new DOMParser();
+    const xmlDoc = parser.parseFromString(text, 'text/xml');
+    const extents = xmlDoc.getElementsByTagName('Extent');
```

Comment:
Select the extent tags by finding the Layer tags and filtering the names that are needed and then loading the extents from there. Utilize an object with keys of layernames and values for the list of dates.